test/integration/standard_filter_test.rb

```diff
@@ -754,6 +754,53 @@ def test_where_array_of_only_unindexable_values
     assert_nil @filters.where([nil], "ok")
   end
 
+  def test_all_filters_never_raise_non_liquid_exception
+    test_drop = TestDrop.new
+    test_drop.context = Context.new
+    test_enum = TestEnumerable.new
+    test_enum.context = Context.new
+    test_types = [
+      "foo",
+      123,
+      0,
+      0.0,
+      -1234.003030303,
+      -99999999,
+      1234.38383000383830003838300,
+      nil,
+      true,
+      false,
+      TestThing.new,
+      test_drop,
+      test_enum,
+      ["foo", "bar"],
+      { "foo" => "bar" },
+      { foo: "bar" },
+      [{ "foo" => "bar" }, { "foo" => 123 }, { "foo" => nil }, { "foo" => true }, { "foo" => ["foo", "bar"] }],
+      { 1 => "bar" },
+      ["foo", 123, nil, true, false, Drop, ["foo"], { foo: "bar" }],
+    ]
+    errors = []
+    test_types.each do |first|
+      test_types.each do |other|
+        (@filters.methods - Object.methods).each do |method|
+          arg_count = @filters.method(method).arity
+          arg_count *= -1 if arg_count < 0
+          inputs = [first]
+          inputs << ([other] * (arg_count - 1)) if arg_count > 1
+          begin
+            @filters.send(method, *inputs)
+          rescue Liquid::ArgumentError => e
+            errors << "#{method} returned #{e.message}"
+          rescue Liquid::ZeroDivisionError => e
+            errors << "#{method} returned #{e.message}"
+          end
+        end
+      end
+    end
+    errors
```

Comment:
This doesn't seem to be used for anything.

Reply:
I can pull these out, however, I might just use this to assert that it doesn't contain an incorrect parameter count error in any response.

Reply:
test my test

Reply:
Not needed as realized this is outside of Strainer so the exception is still thrown.